css/index.css

```diff
@@ -0,0 +1,14 @@
+.row
+width 100%
+backround color: #FB9300
+
+.cards 
+width: 50px;
+height: 100px;
+border 5px, color: #F54748; 
+/* INNER TEXT #LOREM */
+
+.person {
```

Comment:
This ruleset is surrounded correctly with curly braces, but look up how to define an image source in CSS again. The way you linked the picture does not work